GVFS/GVFS.FunctionalTests/Tests/GitCommands/GitCommandsTests.cs

```diff
@@ -45,6 +46,48 @@ public void StatusTest()
             this.ValidateGitCommand("status");
         }
 
+        [TestCase]
+        public void StatusWithGitTraceTest()
+        {
+            Dictionary<string, string> environmentVariables = new Dictionary<string, string>();
+
+            string tracePath = Path.Combine(this.Enlistment.EnlistmentRoot, "trace-log.txt");
+            environmentVariables["GIT_TRACE"] = tracePath;
+            this.ValidateGitCommand(environmentVariables, "status");

```

Comment:
I wonder if this should just be a flag or file name and we set different paths for the vfs repo and control repo so that we can compare the contents of the files against each other?

Reply:
Trace logs have two problems when comparing them directly:

1. Every line starts with a timestamp. We'd need to ignore these.

2. If we verify equality, then we are actually saying "these two commands did the exact same things underneath" which is not always appropriate.

The current setup ignores this by only verifying the path exists and an event that happens exactly once in each execution happens twice in the combined log.

Reply:
I still think keeping 2 separate trace files for the control and vfs repo helps to keep one from affecting the other in unforeseen ways.  Also keeping them separate makes it easier to check the sequence of 
1. File was written
2. Expect it to contain X

Right now the control/vfs repo could have created the file and written the 2 lines that you are counting and the other repo might have done nothing but the test would pass.